packages/shared/src/components/streak/popup/DayStreak.tsx

```diff
@@ -13,34 +15,43 @@ export enum Streak {
 
 interface DayStreakProps {
   streak: Streak;
-  day: number;
+  day?: number;
+  size?: IconSize;
+  className?: string;
   shouldShowArrow?: boolean;
 }
 
 const dayInitial = ['S', 'M', 'T', 'W', 'T', 'F', 'S'];
 const Circle = classed(
   'div',
-  'size-7 rounded-full border border-theme-divider-tertiary',
+  'rounded-full border border-theme-divider-tertiary',
 );
 
 export function DayStreak({
   streak,
   day,
+  size = IconSize.Medium,
+  className,
   shouldShowArrow,
 }: DayStreakProps): ReactElement {
   const renderIcon = () => {
     if (streak === Streak.Completed || streak === Streak.Pending) {
       return (
         <ReadingStreakIcon
           secondary={streak === Streak.Completed}
-          size={IconSize.Medium}
+          className={className}
+          size={size}
         />
       );
     }
 
     return (
       <Circle
```

Comment:
Should we maybe not use this as a Icon in this case?
Because it feels like we are abusing icon props to make it appear as one now?

Reply:
But this is an actual icon that renders an svg file.